DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -129,39 +108,44 @@ juju config kratos-external-idp-integrator \
   microsoft_tenant_id=<tenant_id>
 ```
 
-After a while the *kratos-external-idp-integrator* status will change to active, this means that Azure AD has been added as a Kratos sign in provider.
+> See more: [Charmhub | Kratos External Idp Integrator](https://charmhub.io/kratos-external-idp-integrator)
+
+After a while the kratos-external-idp-integrator status will change to active, this means that Azure AD has been added as a Kratos sign in provider. Congratulations, your deployment now uses Azure AD as your external identity provider.
```

Comment:
Comma splice. Replace the comma with a full stop (or a semi-colon). (Also use code formatting for code names and use a dash between "sign" and "in" and give content piecemeal -- "the status of x" is a better way to focus the attention on what matters here.)

> After a while the status of the `kratos-external-idp-integrator` charm will change to active. This means that Azure AD has been added as a Kratos sign-in provider.